slidedeck/.babelrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  presets: [
+    [
+      'babel-preset-env',
+      {
+        targets: {
+          browsers: ['>0.25%', 'not ie 11', 'not op_mini all']
```

Comment:
No IE 11? What's preventing that?

Reply:
Thought leading ™️ 

https://jamie.build/last-2-versions